src/services/widgets.data.service.ts

```diff
@@ -31,15 +32,22 @@ export const fetchWidgets = async (id: number, yearAgo?: number): Promise<ILocat
 };
 
 function processWidgetsFetchResponse(response: any) {
-  console.log(response);
   const result = {
     meta: response.data.meta,
-    widgets: getVerifiedWidgetsData(response.data.widgets),
+    widgets: response.data.widgets,
   };
+  if (showOnlyOperCards) {
+    result.widgets = getOperWidgetData(response.data.widgets);
+  }
+  result.widgets = getVerifiedWidgetsData(result.widgets);
   result.widgets = addWidgetsVariants(result.widgets);
   return result;
 }
 
+function getOperWidgetData(widgets: Array<any>) {
+  const widgetsToShow = widgets.filter((widget) => operationalCards.includes(widget.name));
```

Comment:
can be returned in the same line
`return widgets.filter((widget) => operationalCards.includes(widget.name));`

Reply:
fixed